packages/evm/contracts/registry/GuardiansRegistry.sol

```diff
@@ -63,9 +67,18 @@ contract GuardiansRegistry is ControlledRecoverable, IGuardiansRegistry, ERC900,
         uint256 lockedBalance;                      // Maximum amount of tokens that can be slashed based on the guardian's drafts
         uint256 availableBalance;                   // Available tokens that can be withdrawn at any time
         uint64 withdrawalsLockTermId;               // Term ID until which the guardian's withdrawals will be locked
+        DeactivationLocks deactivationLocks;        // Guardian's deactivation locks
         DeactivationRequest deactivationRequest;    // Guardian's pending deactivation request
     }
 
+    /**
+    * @dev Guardians can define lock managers to prevent them from deactivating tokens from the registry
+    */
+    struct DeactivationLocks {
```

Comment:
I wonder if we should re-orient this concept as an "activation lock" instead? Sounds a bit weird to base on it on deactivation since you're committing to keep an amount locked via the manager.

Reply:
Sounds better! :)